examples/gnn_explainer_link_pred.py

```diff
@@ -0,0 +1,126 @@
+import os.path as osp
```

Comment:
Can we add a link to this example [here](https://github.com/pyg-team/pytorch_geometric/blob/master/README.md?plain=1#L300)?